containers/podman-desktop
# chore: light dark mode for banners

chore: light dark mode for banners

### What does this PR do?

* Adds new backgrounds for AI lab and Red Hat pack
* Gets the light and dark mode backgrounds and sets the appropriately

### Screenshot / video of UI



Dark:
![Screenshot 2024-08-02 at 1 57 42 PM](https://github.com/user-attachments/assets/b4356818-1ce0-4389-a68b-7b4649e5211d)
![Screenshot 2024-08-02 at 1 58 34 PM](https://github.com/user-attachments/assets/c46a54e1-1a28-43b0-877e-7e4045504304)



Light:
![Screenshot 2024-08-02 at 1 57 50 PM](https://github.com/user-attachments/assets/e3e95b7c-1a8e-46e1-b798-4e571981387b)
![Screenshot 2024-08-02 at 1 58 27 PM](https://github.com/user-attachments/assets/4da8fab6-b5f3-4cc8-9660-e1ccbc04b9d3)






### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop/issues/8140

### How to test this PR?




1. View dashboard
2. Switch themes
3. See new background on banners

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
